### ISSUE TITLE
Merkaartor: recent incompatibility with Qt < 4.5

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#3026]
**[Submitted to the original trac issue database at 9.09pm, Tuesday, 1st June 2010]**

It may be intentional or not, but some recent changes brought an incompatibility with Qt versions smaller than 4.5.

The project file does not check for a specific version, so I assume it was by accident.

The function calls of the style:
```
QColorDialog::getColor(BgColor, this, tr("Select Color"), QColorDialog::ShowAlphaChannel);
```

were introduced with Qt 4.5, according to
http://doc.trolltech.com/4.5/qcolordialog.html#getColor

Earlier versions need the simpler alternative function with less arguments.

The attached patch fixes the issue for me with Qt 4.4.3.

**Comment by tomhughes** [comment]
**Author: koying**
**[Added to the original trac issue at 12.25pm, Thursday, 3rd June 2010]**

(In [21565]) FIX : compilation with 4.4.3 (closes #3026; by gibson)

**Comment by tomhughes** [comment]
**Author: Koying**
**[Added to the original trac issue at 12.28pm, Thursday, 3rd June 2010]**

Thanks, I'll trust you on this one.

It's been a while I don't have a Qt 4.4 available

Copy to Bernd because he asked me about Qt 4.4 compatibility at one point...
